converter.go

```diff
@@ -69,8 +55,18 @@ func convertCoordinates(lon, lat float64) (float64, float64) {
     // The coordinates are floats, so we must convert them into strings to include them in the cs2cs command
     lonStr := strconv.FormatFloat(lon, 'f', -1, 64)
     latStr := strconv.FormatFloat(lat, 'f', -1, 64)
+    
+    var cs2csProjection string
+
+    // Check whether coordinates are NAD83 or Web Mercator. NAD83 will have a positive longitude given our region of the globe.
```

Comment:
We should mention this in the README so that in the future we can easily understand what coordinate systems are supported.